bindings/src/test/java/io/pravega/storage/extendeds3/ExtendedS3StorageTest.java

```diff
@@ -80,6 +84,23 @@ public void testCreateIfNoneMatch() {
     }
     //endregion
 
+    @Test
```

Comment:
Suggest to add case to ensure the tailing space could also be trimmed.

Reply:
Added test